src/simcore_service_deployment_agent/docker_registries_watcher.py

```diff
@@ -46,34 +46,43 @@ def __init__(self, app_config: dict, stack_cfg: ComposeSpecsDict):
         # get all the private registries
         self.private_registries = app_config["main"]["docker_private_registries"]
         # get all the images to check for
-        self.watched_repos = []
+        self.watched_docker_images = []
         if "services" in stack_cfg:
             for service_name in stack_cfg["services"].keys():
                 if "image" in stack_cfg["services"][service_name]:
                     image_url = stack_cfg["services"][service_name]["image"]
-                    self.watched_repos.append({"image": image_url})
+                    self.watched_docker_images.append({"image": image_url})
+                else:
+                    raise RuntimeError(
```

Comment:
conceptually I'm not sure that is a runtime error but more of a value error in the file no?